tensorflow/core/kernels/mkl_matmul_op_fused.cc

```diff
@@ -301,7 +295,7 @@ class MklFusedMatMulOp : public MklDnnMatMulOpBase<T, T> {
       Name("_MklFusedMatMul")                                  \
           .Device(DEVICE_CPU)                                  \
           .TypeConstraint<type>("T")                           \
-          .Label(mkl_op_registry::kMklLayoutDependentOpLabel), \
+          .Label(mkl_op_registry::kMklNameChangeOpLabel),      \
```

Comment:
magic name: kMklNameChangeOpLabel

Reply:
If you don't know how to define it, please give some annotation for it.

Reply:
Intel-tf designs these different kernel type, here have some explanation of  this two kind of kernel type.
``` c++
// MKL operators whose kernels are registered with 'MklLayoutDependentOp' label
// (e.g., MklConv2D) understand input tensors in MKL layout. These operators
// get additional meta-tensors for actual input tensors.
static const char* kMklLayoutDependentOpLabel = "MklLayoutDependentOp";
static const char* kMklLayoutDependentOpLabelPattern =
    "label='MklLayoutDependentOp'";
// MKL operators whose kernels are registered with 'MklNameChangeOp' label
// (e.g., MklMatMul, MklTranspose) do not understand input tensors in MKL
// layout. These operators do not get additional meta-tensors. The signatures of
// these operators are the same as the original TensorFlow operators that they
// correspond to. So these ops just go through a name change during graph
// rewrite pass.
static const char* kMklNameChangeOpLabel = "MklNameChangeOp";
static const char* kMklNameChangeOpLabelPattern = "label='MklNameChangeOp'";
static const char* kMklQuantizedOpLabel = "QuantizedMklOp";
static const char* kMklQuantizedOpLabelPattern = "label='QuantizedMklOp'";
```

Reply:
Sure!